curriculum/challenges/english/25-front-end-development/quiz-css-grid/66ed8fedf45ce3ece4053eb3.md

```diff
@@ -548,3 +548,444 @@ Which of the following is the correct way to use the `minmax()` function?
 }
 ```
 
+## --quiz--
+
+### --question--
+
+#### --text--
+
+Which property controls the alignment of grid items along the inline (row) axis?
+
+#### --distractors--
+
+`align-content`
+
+---
+
+`place-items`
+
+---
+
+`grid-align`
+
+#### --answer--
+
+`justify-items`
+
+### --question--
+
+#### --text--
+
+What does the `grid-auto-rows` property control?
+
+#### --distractors--
+
+The height of explicitly defined rows.
+
+---
+
+The maximum width of grid columns.
+
+---
+
+The spacing between rows.
+
+#### --answer--
+
+The size of implicitly created rows.
+
+### --question--
+
+#### --text--
+
+Which property would you use to make a grid item span multiple rows?
+
+#### --distractors--
+
+`grid-row-span`
+
+---
+
+`row-span`
+
+---
+
+`span-rows`
+
+#### --answer--
+
+`grid-row`
+
+### --question--
+
+#### --text--
+
+What is the purpose of the `place-content` shorthand property?
+
+#### --distractors--
+
+To position the grid container within its parent.
+
+---
+
+To align all items to the center of their cells.
+
+---
+
+To set both `grid-template-rows` and `grid-template-columns`.
+
+#### --answer--
+
+To align the grid content along both block and inline axes.
+
+### --question--
+
+#### --text--
+
+Which value for `grid-auto-flow` would make new items fill columns first?
+
+#### --distractors--
+
+`row`
+
+---
+
+`dense`
+
+---
+
+`row dense`
+
+#### --answer--
+
+`column`
+
+### --question--
+
+#### --text--
+
+What does `align-content: space-between` do in a grid container?
+
+#### --distractors--
+
+Centers all items with equal space around them.
+
+---
+
+Distributes items to start at the beginning and end of the container.
+
+---
+
+Stretches items to fill the container height.
+
+#### --answer--
+
+Distributes extra space evenly between grid tracks.
```

Comment:
same issue here

was this covered in the lectures or workshop? I haven't gone through the lectures, one by one. But it is not on the review page. So either the review page needs to be updated, which is a separate issue. Or we need to update this question to something else.